backend/app/schemas/content.py

```
@@ -1,8 +1,21 @@
 import uuid
+import json
```

Ruff (0.11.9)

`json` imported but unused

Remove unused import: `json`

(F401)